lib/depends.ps1

```diff
@@ -61,6 +61,9 @@ function script_deps($script) {
     if($script -like '*Expand-7zipArchive *' -or $script -like '*extract_7zip *') {
         $deps += '7zip'
     }
+    if($script -like '*Expand-ZstdArchive *' -or $script -like '*extract_zstd *') {
```

Comment:
There is no function `extract_zstd` and no reason to add it

Reply:
ok